sunpy/visualization/visualization.py

```diff
@@ -36,6 +36,22 @@ def fn_itoggle(*args, **kwargs):
 
 
 def axis_labels_from_ctype(ctype, unit):
+    """
+    A function to return axis labels for the given coordinate type and unit
+    using python dictionary.
+    
+    Parameters
+    ----------
+    ctype: string
```

Comment:
`string` should be 

```
`str`
```

This is to ensure that a link is created in our documentation.